Issue1479 Chiller staging subsequences: Efficiency and failsafe condition

milicag opened the pull request:
Closes #1479.

Review after merging [PR#1454](https://github.com/lbl-srg/modelica-buildings/pull/1454) and [PR#1477](https://github.com/lbl-srg/modelica-buildings/pull/1477).

milicag commented:
@AntoineGautier I plan to run pre-review tests as soon as we merge [conf. and status PR](https://github.com/lbl-srg/modelica-buildings/pull/1454).

milicag commented on Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Validation/FailsafeCondition.mo:
```diff
@@ -0,0 +1,136 @@
+    annotation (Placement(transformation(extent={{80,-60},{100,-40}})));
+
+equation
+
```
remove empty line

milicag commented on Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Validation/EfficiencyCondition.mo:
```diff
@@ -0,0 +1,57 @@
+
+  Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Subsequences.EfficiencyCondition
+    effCon
+    "Failsafe condition to test for the operating part load ratio input"
```
move to previous line

milicag commented:
@AntoineGautier This is ready for review as soon as the tests pass.

milicag commented:
@JayHuLBL This is ready for review.

JayHuLBL commented on Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/EfficiencyCondition.mo:
```diff
@@ -0,0 +1,81 @@
+  "Efficiency condition used in staging up and down"
+
+  parameter Modelica.SIunits.Time delayStaCha = 300
+  "Enable delay";
```
should it be 15 minutes (900 s)?

JayHuLBL commented on Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/EfficiencyCondition.mo:
```diff
@@ -0,0 +1,82 @@
+  "Efficiency condition used in staging up and down"
+
+  parameter Modelica.SIunits.Time delayStaCha = 300  "Enable delay";
+
```
should it be 15 minutes?

milicag replied:
yes, corrected.

JayHuLBL commented:
@milicag checked the latest RP1711 and there is no update on this sequence. Besides the one I marked, I have no other comment. 

It should be ready to merge after addressing the comment and CI tests passed.

milicag commented:
@JayHuLBL I addressed your comment. In the header of the staging table for the newest version there is a conditional for series chillers plants, so I added that in the failsafe sequence and tests. I also added in the info section the 1711 version. When the tests pass please take a quick look before merge due to the added conditional.
